javacutil/src/org/checkerframework/javacutil/trees/TreeBuilder.java

```diff
@@ -63,6 +63,18 @@ public TreeBuilder(ProcessingEnvironment env) {
     }
 
     /**
+     * Copy an AST Tree.
+     *
+     * @param target a tree to be copied
+     * @return a copied tree
+     */
+    @SuppressWarnings("unchecked")
+    public <T extends Tree> T copy(T target) {
```

Comment:
Why is the parameter called `target` and not `source` or `input`? What is it the target of?